### PR TITLE
Readying for a 0.1.0

### PR DESCRIPTION
**Pull request by stahnma**
* Adding a travis.yml file
* Added CHANGLOG and CONTRIBUTING.md
* Minor tweak to take gemspec under Puppet Labs name.

### CODE REVIEW

**Comment by stahnma** [comment]
:boom: travis is now working.

**Comment by shrug** [comment]
:+1:

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+language: ruby
+bundler_args: --without development
+script: "bundle exec rake"
+notifications:
+  email: false
+rvm:
+  - 2.2.0
+  - 2.1.5
+  - 2.1.0
```
Comment (mckern) [inline .travis.yml]: 2.1.5 should be sufficient?

**Comment by mckern** [comment]
:+1: for following the doc I wrote!

File: .travis.yml (hunk 1 above)
Comment (stahnma) [inline .travis.yml]: mo' thorough is mo' better? 

I dunno. Happy to take one of those out if you think it matters.

Comment (mckern) [inline .travis.yml]: Might cut down on Travis run-time.

**Pull request merged**